Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -613,10 +647,52 @@ private async void CheckForIdlePlayback(object state)
                 playingSessions = Sessions.Where(i => i.NowPlayingItem is not null)
                     .ToList();
             }
+            else
+            {
+                StopIdleCheckTimer();
+            }
+        }
+
+        private async void CheckForInactiveSteams(object state)
+        {
+            var pausedSessions = Sessions.Where(i =>
+                (i.NowPlayingItem is not null) &&
+                (i.PlayState.IsPaused == true) &&
+                (i.LastPausedDate is not null)).ToList();
```

Comment:
```suggestion
            var pausedSessions = Sessions
                .Where(i =>
                    i.NowPlayingItem is not null
                    && i.PlayState.IsPaused
                    && i.LastPausedDate is not null)
                .ToList();
```

Reply:
It this personal preference or referenced by a code style guide regarding the indentation? Because while looking through the code (both Emby and Jellyfin packages) there is no clear style for a case like this.

I agree with your points, but would prefer to keep `.Where(i =>` in line 658 together with `Sessions`.

Reply:
The proposed style changes is what we try to use in new code. It's not documented anywere though, just preference.

Reply:
Correct- it's just preference. Moving `.Where` to L658 is fine by me